### ISSUE TITLE
Remote processing of zip files

### ISSUE THREAD

**Issue by publcif** [jamesrhester/ImgCIFHandler.jl#8]
For servers that support range headers in requests (Range: bytes=...) it is possible to extract parts of a zip archive remotely.
A python script that does this is at https://github.com/gtsystem/python-remotezip

I have tested this 'out-of-the-box' and think it may well be very useful in this project (see fork at https://github.com/publcif/ImgCIFHandler.jl)

**Comment by jamesrhester** [comment]
Yes, this is something we could easily re-implement in Julia (only 150 lines in Python, should be possible), or else use the Julia 'pycall' module to call the remotezip class directly. My streams/buffers/web programming is not that good so might take a little messing around.

**Comment by publcif** [comment]
That sounds great.

**Comment by jamesrhester** [comment]
Fixed in d8666ba. Zip files are now among the most economical to deal with in terms of downloads as only the particular frame requested is downloaded (if the remote server supports range requests, which all of the major repositories seem to).
